### PR TITLE
Common dao changes

### PR DESCRIPTION
**Pull request by AlexKazmirchuk**

### CODE REVIEW

File: softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml [hunk 1]
```diff
@@ -60,5 +60,18 @@
             </sql>
         </rollback>
     </changeSet>
+    
+    <changeSet id="contacts-table-creating-4" author="AlexKazmirchuk">
+        <createTable tableName="contacts">
+            <column name="id" type="varchar(255)">
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml]: @AlexKazmirchuk id 255 ?

File: softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml [hunk 2]
```diff
@@ -60,5 +60,18 @@
...
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(255)"/>
+            <column name="phone" type="varchar(255)"/>
+            <column name="profile_id" type="varchar(255)">
```
Comment (Dominys) [inline softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml]: @AlexKazmirchuk profile_id 255 ?

File: softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml (hunk 1 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml]: @Dominys fixed

File: softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml (hunk 2 above)
Comment (AlexKazmirchuk) [reply softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml]: @Dominys fixed (I know I should not change change sets ,but when I will have all tables I will add new change sets instead changing old)

**Review by Dominys — APPROVED** [review]

**Pull request merged**